templates/gameboard.html

```diff
@@ -142,4 +172,84 @@ <h4 class="collapsible" id="suggested-queries-header">Suggested Queries</h4>
+                </div>
+                <div id="verify-detection-form" class="row-simple" style="width:100%">
+                    <div class="column" style="width:75%;margin-bottom:-35px">
+                        <div class="column" style="flex:60%">
+                            <table frame=void rules=rows style="border-spacing:2px;width:100%">
+                                <tr>
+                                    <td style="width:10%"><b style="color:white">host:</b></td>
+                                    <td>
+                                        <select id="host-select" style="width:99%; margin:5px">
+                                            <option value="" disabled="disabled" selected>Select a host</option>
+                                            {% for host in hosts %}
+                                                <option id="select-{{ host }}" value="{{ host }}">{{ host }}</option>
+                                            {% endfor %}
+                                        </select>
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td style="width:10%"><b style="color:white">tactic:</b></td>
+                                    <td>
+                                        <select id="tactic-select" onchange="verifyPopulateTechniques('verify-detection-modal', tactics_and_techniques)" style="width:99%; margin:5px">
+                                            <option disabled selected>Choose a tactic</option>
+                                            {% for tactic in tactics %}
+                                                <option value="{{ tactic }}" data-tactic="{{ tactic }}">{{ tactic }}</option>
+                                            {% endfor %}
+                                        </select>
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td style="width:10%"><b style="color:white">technique:</b></td>
+                                    <td>
+                                        <select id="technique-select" style="width:99%; margin:5px">
+                                            <option disabled selected>Choose a technique</option>
+                                        </select>
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td style="width:10%">
+                                        <div class="toggle" style="margin-top:3px">
+                                        <label class="switch"><input type="checkbox" id="togBtnHunt">
+                                            <div class="slider round" style="background-color:black">
+                                                <span class="on" style="top:45%"><b style="font-size:16px">guid</b></span>
```

Comment:
consider addding a tooltip / duk that explains that for the guid option to work, a blue agent must have been running during the operation

Reply:
Agreed. Sounds good 👍